runtimevar/blobvar/blobvar.go

```diff
@@ -16,6 +16,19 @@
 // variables read from a blob.Bucket.
 // Use NewVariable to construct a *runtimevar.Variable.
 //
+// URLs
+//
+// For runtimevar.OpenVariable URLs, blobvar registers for the scheme "blob".
+// The host+path is used as the blob key. blobvar supports the following URL
+// parameters:
+//   - bucket: The URL to be passed to blob.OpenBucket (required).
+//       blob.OpenBucket will be called once per unique bucket URL.
+//   - decoder: The decoder to use. Defaults to runtimevar.BytesDecoder.
+//       See runtimevar.DecoderByName for supported values.
+//   - wait: The poll interval; supported values are from time.ParseDuration.
+//       Defaults to 30s.
+// Example URL: "blob://myvar.txt?bucket=fileblob:///foo/bar&decoder=string".
```

Comment:
Scheme is incorrect here:

```suggestion
// Example URL: "blob://myvar.txt?bucket=file:///foo/bar&decoder=string".
```